### DISCUSSION TITLE
What element has the chemical symbol **K**?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#255] [Galaxy Trivia] [answered]
**Hint:** 🍌🥑🍉 pack more of it.

**Comment by live627** [comment, marked as answer]
Potassium

**Reply by ParanoidUser** [reply]
Bingo! Congratulations on your well deserved [Galaxy Brain](https://github.com/live627?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
